.github/workflows/ci.yml

```diff
@@ -17,11 +17,18 @@ jobs:
     strategy:
       matrix:
         python-version: ["3.8", "3.9", "3.10"]
-
     steps:
       - name: Checkout code
         uses: actions/checkout@v2
 
+      - name: Setup SSH key
+        uses: webfactory/ssh-agent@v0.6.0
+        with:
+          ssh-private-key: ${{ secrets.DXTB_SSH_PRIVATE_KEY }}
+
+      - name: Checkout submodules
+        run: git submodule update --init --recursive
+
```

Comment:
This is undoing changes made previously which removed the ssh key that was causing issues so we should delete this block here.